tools/metricsdocs/metricsdocs.go

```diff
@@ -42,18 +42,20 @@ func writeToFile(metricsList metricList) {
 type metric struct {
 	name        string
 	description string
+	mType       string
```

Comment:
If the `m` prefix for this field stands for "metric", I would say it is redundant, since the struct named `metric` so it is obvious the type of the metric. Same comment for:
https://github.com/kubevirt/hyperconverged-cluster-operator/pull/1830/files#diff-4bc91a74bc47b8467cdcd25a7e3fe1651dbe44907f478377eca31b982f444f23R20

Reply:
I added this prefix to avoid confusion with Golang's saved word - `type`.
Do you have any suggestions for this field?

Reply:
yes, you're right. we don't want to export this field by capitalizing it, and using `_type` instead won't improve much.